mod/filer.php

```diff
@@ -15,7 +15,7 @@ function filer_content(App $a)
 		killme();
 	}
 
-	$term = unxmlify(trim($_GET['term']));
+	$term = unxmlify(isset($_GET['term']) ? trim($_GET['term']) : null);
```

Comment:
Later in the code `$term` string length is checked, please initialize with an empty string instead. You can also wrtie it with `defaults()` like
```php
$term = unxmlify(trim(defaults($_GET, 'term', '')));
```

Reply:
That code looks much better than my "bare".